diff --git a/CMakeLists.txt b/CMakeLists.txt
index a4f2f8e..a5149dc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -52,14 +52,20 @@ add_custom_target(extract-messages ${CMAKE_SOURCE_DIR}/scripts/extract-messages
 # Build Translations
 CREATE_TRANSLATION_TARGETS()
 
-if(APPLE OR WIN32)
-    install(DIRECTORY UM DESTINATION lib/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages)
-else()
-    install(DIRECTORY UM DESTINATION lib/python${PYTHON_VERSION_MAJOR}/dist-packages)
+if(NOT PYTHON_SITE_PACKAGES_DIR)
+    if(APPLE OR WIN32)
+        set(PYTHON_SITE_PACKAGES_DIR lib/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages
+            CACHE STRING "Directory to install Python bindings to")
+    else()
+         set(PYTHON_SITE_PACKAGES_DIR lib/python${PYTHON_VERSION_MAJOR}/dist-packages CACHE
+            STRING "Directory to install Python bindings to")
+    endif()
 endif()
+
+install(DIRECTORY UM DESTINATION ${PYTHON_SITE_PACKAGES_DIR})
 install(FILES ${CMAKE_SOURCE_DIR}/cmake/UraniumTranslationTools.cmake
-        DESTINATION ${CMAKE_INSTALL_DATADIR}/cmake-${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}/Modules/ )
+        DESTINATION ${CMAKE_INSTALL_DATADIR}/cmake/Modules/ )
 install(DIRECTORY resources DESTINATION ${CMAKE_INSTALL_DATADIR}/uranium)
-install(DIRECTORY plugins DESTINATION lib/uranium)
+install(DIRECTORY plugins DESTINATION ${CMAKE_INSTALL_LIBDIR}/uranium)
 
 include(CPackConfig.cmake)
